plugins/modules/clustering/nomad/nomad_job.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: FERREIRA Christophe (@chris93111)
+version_added: "1.2.0"
+short_description: Launch a Nomad Job
+description:
+    - Launch a Nomad job.
+    - Stop a Nomad job.
+    - Force start a Nomad job
+requirements:
+  - python-nomad
+options:
+    host:
+      description:
+        - FQDN of Nomad server.
+      required: true
+      type: str
+    use_ssl:
+      description:
+        - Use TLS/SSL connection.
+      type: bool
+      default: false
```

Comment:
```suggestion
      default: true
```
Default values for options should be secure by default.